src/shared/services/config/posthog-config.ts

```diff
@@ -1,30 +1,60 @@
 export interface PostHogClientConfig {
+	/**
+	 * The main API key for PostHog telemetry service.
+	 */
 	apiKey?: string | undefined
+	/**
+	 * The API key for PostHog used only for error tracking service.
+	 */
 	errorTrackingApiKey?: string | undefined
 	host: string
 	uiHost: string
 }
 
+/**
+ * Helper type for a valid PostHog client configuration.
+ * Must contains api keys for both telemetry and error tracking.
+ */
 export interface PostHogClientValidConfig extends PostHogClientConfig {
 	apiKey: string
 	errorTrackingApiKey: string
 }
 
-// Public PostHog key (safe for open source)
-const posthogProdConfig = {
-	apiKey: "phc_qfOAGxZw2TL5O8p9KYd9ak3bPBFzfjC8fy5L6jNWY7K",
-	errorTrackingApiKey: "phc_qfOAGxZw2TL5O8p9KYd9ak3bPBFzfjC8fy5L6jNWY7K",
+/**
+ * Public PostHog keys for prod and dev (safe for open source).
+ * NOTE: Soon to be deprecated and replaced with environment variables.
+ */
+const PUBLIC_POSTHOG_API_KEYS = {
+	PROD: "phc_qfOAGxZw2TL5O8p9KYd9ak3bPBFzfjC8fy5L6jNWY7K",
+	DEV: "phc_uY24EJXNBcc9kwO1K8TJUl5hPQntGM6LL1Mtrz0CBD4",
```

Comment:
Before POSTHOG_API_KEY and POSTHOG_ERROR_API_KEY are added to the GitHub secret storage, we could continue use these public keys

Reply:
Removed